pkg/yurtmanager/controller/yurtappoverrider/yurtappoverrider_controller.go

```diff
@@ -91,6 +100,15 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 	if err != nil {
 		return err
 	}
+	go wait.Until(func() {
+		overriderList := &appsv1alpha1.YurtAppOverriderList{}
+		if err := r.(*ReconcileYurtAppOverrider).Client.List(context.TODO(), overriderList); err != nil {
+			klog.Errorf("failed to list yurtappoverriders:%+v", err)
+		}
+		for _, overrider := range overriderList.Items {
+			mgr.GetEventRecorderFor(names.YurtAppOverriderController).Event(overrider.DeepCopy(), "", "", "")
+		}
+	}, 5*time.Minute, nil)
```

Comment:
why need to report events for each yurtappoverrider every 5 minutes?

Reply:
I made a mistake about EventRecorder. 
As a correction, I didn't find the request adding interface of workqueue, so I modified it to call the reconcile function directly, I don't know whether it's appropriate or not.